### DISCUSSION TITLE
Quick Reblog: Hide native "reblog now" button

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#843] [Suggestion Box]
This seems like an ultimately fairly bad idea to me, but I figured I'd bring it up in case someone has a strong argument for it:

Quick Reblog being enabled makes the "reblog now" button in the native quick reblog popup kind of redundant, and that's the one under your cursor when you click reblog. Hiding it when Quick Reblog is enabled would make the "regular reblog" method clicking twice instead of clicking once, carefully moving your cursor down a bit, then clicking again.

**Comment by AprilSylph** [comment]
As a reminder, the native quick reblog is an experiment right now, and I don't think it's being A/B tested right now. Our main accounts are both opted into the test permanently for XKit conflict resolution purposes.

**Reply by marcustyphoon** [reply]
Oh! Gotcha, I didn't think of that. That does explain why I saw some people lamenting its vanished-ness and went, "weird, still shows up for me."
